code-challenges.js

```diff
@@ -26,7 +26,48 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 17
 
-// Explain your code:
+
+
+
+// CREATE a function that takes in an array of numbers
+// OUTPUTS a total sum of all the numbers in an array added together
+const totalWorldSeriesRuns = (arrayOfNumbers) => {
+
+// DECLARE a variable called totalNumberOfHits and set equal to zero
+  let totalNumberOfHits = 0;
+
+  // ITERATE through the array adding all the values together
+  for(i=0; i<arrayOfNumbers.length; i++){
+
+  // REDECLARE totalNumberOfHits to plus equals the value of the index
+      totalNumberOfHits += arrayOfNumbers[i];
+  }
+  // RETURN totalNumberOfHits
+  return totalNumberOfHits;
+}
+console.log(totalWorldSeriesRuns(padres1984WorldSeriesRuns));// outputs: 15
+console.log(totalWorldSeriesRuns(padres1998WorldSeriesRuns));//outputs 17
+
+
+
+// CREATE a function inputting an array of numbers
+// OUTPUT the total of the array added together
+const totalWorldSeriesRuns1 = (arrayOfNumbers) => {
+
+  // ADDING the value of each index to eachother by the total length of the array
+  return arrayOfNumbers.reduce((total, value) => total + value);
+  // RETURN the array reduced to a single value
+}
+
+console.log(totalWorldSeriesRuns1(padres1984WorldSeriesRuns));//outputs: 15
+console.log(totalWorldSeriesRuns1(padres1998WorldSeriesRuns));// outputs: 17
+
+
+
+// Explain your code: I Created a function called totalWorldSeriesRuns that utilizes a for loop to iterate through the array adding each value of the array toghether to total the sum of the values of the array. Redeclaring a variable I originally set to zero called totalNumberOfHits. Using the plus equals sign to add each value of the index to the next value in the next index. Returning the total number of values across the total number of indexes in the array, combining them into the variable totalNumberOfHits. This can also be done using the .reduce method. By creating your function then returning your main functions parameter.reduce. Declaring the parameters within .reduce to equal the total number of indexes within the array and the value of each index in the array. Internally creating another function that takes the total number of indexes and adds the values of each index together reducing the array to a single value. I plan on researching the plus equals and .reduce to better understand how and why they work.
+
```

Comment:
Matt, this is AWESOME! great pseudocode, clean code with good logic.  I LOVE that you also worked it using .reduce!  Your desire to learn is fun to experience.  Great work!